pages/common/hg-serve.md

```diff
@@ -0,0 +1,27 @@
+# hg serve
+
+> Start a standalone Mercurial web server for browsing repositories.
+
+- Start a web server instance:
+
+`hg serve`
+
+- Start a web server instance on the specified port:
+
+`hg serve --port {{port}}`
+
+- Start a web server instance on the specified listening address:
+
+`hg serve --address {{address}}`
+
+- Start a web server instance with a specific name:
+
+`hg serve --name {{name}}`
+
+- Start a web server instance using the specified theme:
+
+`hg serve --style {{style}}`
+
+- Start a web server instance using the specified SSL certificate:
```

Comment:
Where would one specify the ssl key ?

Reply:
It looks like it needs to be a bundled `.PEM` file containing both the key and the certificate.